MdePkg/Include/Register/Intel/ArchitecturalMsr.h

```diff
@@ -517,7 +517,7 @@ typedef struct {
   ///
   /// Pad header so total size is 2KB
   ///
-  UINT8     Reserved[SIZE_2KB - 8 * sizeof (UINT32)];
+  UINT8     Reserved[SIZE_1KB - 8 * sizeof (UINT32)];  // MU_CHANGE - Reduce padding

```

Comment:
Correct me if I am missing something, but I think we should leave this intact?

Reply:
Yeah for binary compatibility the Software STM header needs to start at the same offset (2k).

Reply:
I reverted the change based on discussions.